pkg/chunked/storage_linux.go

```diff
@@ -819,11 +848,13 @@ func (d *chunkedZstdDiffer) ApplyDiff(dest string, options *archive.TarOptions)
 				return output, err
 			}
 			dstFile.Close()
```

Comment:
So the hard-link code ignores the new-container file attributes entirely? That feels like something that _must_ be documented while we still remember.

Reply:
Still outstanding I think.